sbt/src/main/scala/io/radicalbit/Job.scala

```diff
@@ -0,0 +1,19 @@
+package io.radicalbit
```

Comment:
Create a more specified package to reflect the framework and the kind of snippet i.e `io.radicalbit.flink.batch`